jakartaee9/src/web-partialresponse_3_0.xsds

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
 
-    Copyright (c) 2018 Oracle and/or its affiliates. All rights reserved.
+    Copyright (c) 2020 Oracle and/or its affiliates. All rights reserved.
```

Comment:
You know what I'm going to say, right?  :-)
Please check all files to make sure the first copyright date has not been lost.